pipelines.adoc

```diff
@@ -39,25 +39,25 @@ By default, the pipelines run and deploy the application in the `+kabanero+` nam
 
 For more information, see link:https://github.com/kabanero-io/kabanero-pipelines[the kabanero-pipelines repo].
 
-== Running Kabanero pipelines
+== Running Pipelines
 
-Explore how to use Kabanero Pipelines to build and manage Kabanero collections.
+Explore how to use Pipelines to build and manage Collections.
 
 === Prerequisites
 
-. link:https://github.com/kabanero-io/kabanero-foundation[Kabanero foundation] must be installed on Red Hat Origin Community Distribution of Kubernetes (OKD) or the OpenShift Container Platform (OCP) cluster.
+. link:https://github.com/kabanero-io/kabanero-foundation[Kabanero foundation] must be installed on Red Hat Origin Community Distribution of Kubernetes (OKD) or the OpenShift Container Platform (OCP) cluster.  At present Kabanero 0.3.0 and greater only supports OCP 4.2+.  When OKD provides a comparable 4.x stream, Kabanero will again support OKD providing a full open source stack.
 
-. link:https://github.com/tektoncd/dashboard[Tekton Dashboard] is installed by default with Kabanero Foundation. To find the Tekton Dashboard URL, login to your OKD cluster and run the `+oc get routes+` command or check in the OKD console. You can also find the dashboard URL from the Kabanero landing page in the OKD or OCP console.
+. link:https://github.com/tektoncd/dashboard[Tekton Dashboard] is installed by default with Kabanero's Kubernetes Operator. To find the Tekton Dashboard URL, login to your cluster and run the `+oc get routes+` command or check in the console. You can also find the dashboard URL from the Kabanero getting started page in the OKD or OCP console.
 
 . Dynamic volume provisioning or a persistent volume must be configured. See the following section for details.
 
 === Getting started
 
 ==== Set up a persistent volume to run pipelines
 
-Tekton pipelines require a configured volume that is used by the framework to share data across tasks. The `build task`, which uses `Buildah`, also requires a volume mount. The pipelinerun creates a Persistent Volume Claim (PVC) with a requirement for five Gi of persistent volume.
+Pipelines require a configured volume that is used by the framework to share data across tasks. The `build task`, which uses `Buildah`, also requires a volume mount. The pipelinerun creates a Persistent Volume Claim (PVC) with a requirement for five Gi of persistent volume.
 
-. Log in to your cluster. For example, for OKD
+. Log in to your cluster. For example, 
```

Comment:
Is the example still correct as shown if it is not applied to OKD?

these prompts should be rewritten as follows-

Log in to your cluster, as shown in the following example